aws-datastore/src/test/java/com/amplifyframework/datastore/AWSDataStorePluginTest.java

```diff
@@ -66,6 +67,7 @@ public void setup() {
      * @throws JSONException Not expected; on failure to arrange configuration object
      * @throws DataStoreException Not expected; on failure to configure of initialize plugin
      */
+    @Ignore("TODO: need a mechanism to re-enable this scenario........")
```

Comment:
.e.g., we need to inspect state of API category to understand if it has a suitable configuration for our use .... (what are the appropriate checks for this, that don't leak abstraction? In reality, it's gonna break hard if it is not an `AWSApiPlugin`. But in fantasy land,  we want a flat dependency tree where-in plugins _do not_ take a dependency on one another.)